File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/python/cwe-89/code_before2.py
# From cwe-snippets, snippets_2/non-compliant/Python/0046.py

userName = req.field('userName')
emailId = req.field('emaiId')
results = db.emails.find({"$where", "this.owner == \"" + userName + "\" ?? this.emailId == \"" + emailId + "\""});

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/cpp/cwe-89/code_before2.cpp
// From cwe-snippets, snippets_2/non-compliant/C++/0055.cpp

ctx.getAuthUserName(?userName); {
CString query = "SELECT * FROM items WHERE owner = '"
     + userName + "' AND itemname = '"
     + request.Lookup("item") + "'";
dbms.ExecuteSQL(query);